src/components/vm/disks/vmDiskColumns.jsx

```diff
@@ -167,7 +167,7 @@ export const DiskActions = ({ vm, vms, disk, supportedDiskBusTypes, idPrefixRow,
     const Dialogs = useDialogs();
 
     function openMediaInsertionDialog() {
-        Dialogs.show(<AddDiskModalBody idPrefix={idPrefixRow + "-insert-dialog"}
+        Dialogs.show(<AddDiskModalBody idPrefix={idPrefixRow + "-insert-dialog-adddisk"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1303-20231117-134447-13cb53e1-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)